### DISCUSSION TITLE
Where can I find Amoro's recent maven dependencies

### DISCUSSION THREAD

**Discussion by JinsYin** [apache/amoro#2508] [Q&A]
> https://mvnrepository.com/artifact/com.netease.arctic/arctic-core

It seems that there has been no Maven dependency provided since the arctic was renamed to amoro.

**Comment by baiyangtx** [comment]
Yes, due to the fact that the project is being incubated, it may change its name again in the future, so we don't apply for a new Maven Group ID for the time being
